full/Dockerfile

```diff
@@ -213,10 +211,8 @@ RUN curl -fsSL https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-ins
         echo 'eval "$(pyenv init -)"'; \
         echo 'eval "$(pyenv virtualenv-init -)"'; } >> /home/gitpod/.bashrc.d/60-python \
     && pyenv update \
-    && pyenv install 2.7.18 \
     && pyenv install 3.8.6 \
-    && pyenv global 3.8.6 2.7.18 \
-    && python2 -m pip install --upgrade pip \
+    && pyenv global 3.8.6 \
```

Comment:
@AlexTugarev It's cool to no longer install Python 2 via Pyenv, right?

And actually, I'd love to get rid of Pyenv entirely soon (because both Python2 and Python3 are already installed as dependencies in our initial `apt-get install`). But we should probably keep it in place until Full Workspace Backup is enabled for everyone.

Reply:
> But we should probably keep it in place until Full Workspace Backup is enabled for everyone.

I'd love to get rid of it!